libs/ml/include/ml/ops/activations/logsoftmax.hpp

```diff
@@ -39,7 +39,7 @@ class LogSoftmax : public fetch::ml::Ops<T>
   using SizeType      = typename ArrayType::SizeType;
   using VecTensorType = typename Ops<T>::VecTensorType;
 
-  explicit LogSoftmax(SizeType axis = 0)
+  explicit LogSoftmax(SizeType axis = 1)
```

Comment:
we need to check example code for invocations of LogSoftmax that do not rely on the default axis

Reply:
No example is using LogSoftmax